src/chapters/css/exercises.rst

```diff
@@ -0,0 +1,52 @@
+Exercises: CSS
+==============
+
+.. admonition:: Note
+
+   We built a website for you to test your CSS knowledge! To allow you to view
+   your work in its own browser tab, we saved the
+   `starter code in repl.it <https://repl.it/@launchcode/LCHS-CSS-Exercises>`__.
+
+   Complete the exercises, then follow your teacher's instructions about how to
+   submit your work.
+
+Add the following style rules to the website. It's OK to look back in the
+chapter, but try to complete as many of the tasks as possible before you check!
+
+External CSS
+------------
+
+Add element, class, and id selectors to the ``style.css`` file to do the
+following:
+
+#. Change the background color of the whole page to yellow.
+#. Change the text color to green for all paragraphs, and set ``font-family``
+   to ``Courier``.
+#. Change all ``h1`` elements to 48px font size and ``h2`` elements to 36px.
+   Also, underline the ``h2`` text, and overline/underline the ``h1`` text.
```

Comment:
Maybe clarify that you want them to both overline and underline the ``h1`` text

Reply:
Fixed.  Now `h1` gets the underline (bonus for wavy), and `h2` gets a clear call for both an overline and an underline.